services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -3325,7 +3325,7 @@ boolean hasFatalDataValidationError(PushMonitor pushMonitor, String topicName) {
       OfflinePushStatus offlinePushStatus = pushMonitor.getOfflinePushOrThrow(topicName);
       return offlinePushStatus.hasFatalDataValidationError();
     } catch (VeniceException e) {
-      LOGGER.warn("Failed to get offline push status for topic: {}", topicName, e);
+      LOGGER.warn("Failed to get offline push status for topic: {}. It might not exist anymore.", topicName);
```

Comment:
Felix, thanks for fixing the exception logging. I agree that the printout of the stack trace in this call path is unnecessary.

Reply:
Thanks for reviewing!